exec/vector/src/main/java/org/apache/drill/exec/record/metadata/TupleMetadata.java

```diff
@@ -46,6 +46,10 @@
 public interface TupleMetadata extends Propertied, Iterable<ColumnMetadata> {
 
   public static final String IS_STRICT_SCHEMA_PROP = DRILL_PROP_PREFIX + "strict";
+  public static final String HAS_HEADERS_PROP = DRILL_PROP_PREFIX + "headers";
```

Comment:
I am not sure this should be placed here, these properties are mostly related to text readers unlike `IS_STRICT_SCHEMA_PROP` or `TupleMetadata` class itself. Maybe its better to move these properties to separate class which will clearly indicate that they are applicable for the text readers only.